src/shared/validations/validations.ts

```diff
@@ -2,14 +2,35 @@
  * This module sets up and configures validations for processing records through Flatfile.
  */
 
-import { FlatfileListener } from '@flatfile/listener'
-import { plmValidations } from '../../workflows/plm/validations/plmValidations'
+import { FlatfileListener } from '@flatfile/listener';
+import { plmValidations } from '../../workflows/plm/validations/plmValidations';
+import { hcmValidations } from '../../workflows/hcm/validations/hcmValidations';
 
 /**
  * @param {FlatfileListener} listener - The FlatfileListener instance to which custom
  * validations are attached. This object acts as the conduit through which records are
  * received from Flatfile, validated, and potentially modified or flagged for correction.
  */
 export function validations(listener: FlatfileListener) {
-  listener.use(plmValidations)
+  listener.use(plmValidations);
+  listener.use(hcmValidations);
```

Comment:
Can we split these validations into project specific files? Running plm validations against hcm or vice versa is confusing and also not needed

Reply:
yeah that makes sense